Arrowgene.Ddon.GameServer/Handler/CharacterPawnGoldenReviveHandler.cs

```diff
@@ -19,7 +19,7 @@ public CharacterPawnGoldenReviveHandler(DdonGameServer server) : base(server)
         public override void Handle(GameClient client, StructurePacket<C2SCharacterPawnGoldenReviveReq> req)
         {
             S2CCharacterPawnGoldenReviveRes res = new S2CCharacterPawnGoldenReviveRes(req.Structure);
-            res.GoldenGemstonePoint = 120;
+            res.GoldenGemstonePoint = 0; // TODO: Implement
```

Comment:
Do we even need to assign it?

Reply:
If we want people not to have infinite revives, yea.

Reply:
Isn't it zero by default? Anyways, not a big deal.

Reply:
Yes, but the GP menu pcap reverted it to 71. Then you could use a revive and itd go to 120, and therefore be infinite